config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+
+  resources :book
```

Comment:
Here you are opening more route that what your need.
If you want to check what are the routes in your app you can run:
`bundle exec rails routes`